src/workersrc.js

```diff
@@ -24,11 +24,12 @@
 
 import canDownload from './utils/canDownload'
 import redirectIfNotIframe from './utils/redirectIfNotIframe'
+import { loadState } from '@nextcloud/initial-state'
 
 /**
  * Checks if the page is displayed in an iframe. If not redirect to /.
  **/
-redirectIfNotIframe()
+// redirectIfNotIframe()
```

Comment:
TODO: revert before merge, this allows to display the iframe without being redirected to the nc dashboard